src/VolleyManagement.API/Properties/launchSettings.json

```diff
@@ -0,0 +1,29 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:49940/",
+      "sslPort": 44370
```

Comment:
Can we change it to match what we had in mvc?

Reply:
Yes we can